tests/helium-entity-manager.ts

```diff
@@ -136,6 +143,226 @@ describe("helium-entity-manager", () => {
+      await hemProgram.methods.initializeDataOnlyV0({
+        maxDepth: height,
+        maxBufferSize: buffer,
+        treeSpace: new BN(getConcurrentMerkleTreeAccountSize(height, buffer, canopy)),
+        newTreeDepth: height,
+        newTreeBufferSize: buffer,
+        newTreeSpace: new BN(getConcurrentMerkleTreeAccountSize(height, buffer, canopy)),
+        newTreeFeeLamports: new BN((LAMPORTS_PER_SOL * 30) / 2**height),
+        name: "DATAONLY",
+        metadataUrl: "test",
+      }).accounts({
+        dao,
+        merkleTree: merkle.publicKey,
+      }).preInstructions([
+        ComputeBudgetProgram.setComputeUnitLimit({ units: 350000 }),
+      ]).rpc({skipPreflight: true});
+
+      const doAcc = await hemProgram.account.dataOnlyConfigV0.fetch(dataOnlyConfigKey(dao)[0]);
+
+      ({getAssetFn, getAssetProofFn, hotspot} = await createMockCompression({
+        collection: doAcc.collection,
+        dao,
+        merkle: merkle.publicKey,
+        ecc,
+        hotspotOwner,
+      }));
+
+      await dcProgram.methods
+        .mintDataCreditsV0({
+          hntAmount: toBN(startDcBal, 8),
+          dcAmount: null,
+        })
+        .accounts({ dcMint })
+        .rpc({ skipPreflight: true });
+    });
+    it("issues and onboards a data only hotspot", async () => {
+      let hotspotOwner = Keypair.generate();
+      const issueMethod = hemProgram.methods
+        .issueDataOnlyEntityV0({
+          entityKey: Buffer.from(bs58.decode(ecc)),
+        })
+        .preInstructions([
+          ComputeBudgetProgram.setComputeUnitLimit({ units: 500000 }),
+        ])
+        .accounts({
+          recipient: hotspotOwner.publicKey,
+          dao,
+          eccVerifier: eccVerifier.publicKey,
+        })
+        .signers([eccVerifier]);
+
+      const { keyToAsset } = await issueMethod.pubkeys();
+      await issueMethod.rpc({ skipPreflight: true });
+        
+      console.log(keyToAsset?.toString());
+      const ktaAcc = await hemProgram.account.keyToAssetV0.fetch(
+        keyToAsset!
+      );
+      expect(Boolean(ktaAcc)).to.be.true;
+
+      const {
+        args,
+      } = await proofArgsAndAccounts({
+        connection: hemProgram.provider.connection,
+        assetId: hotspot,
+        getAssetFn,
+        getAssetProofFn,
+      });
+      const onboardMethod = hemProgram.methods
+        .onboardDataOnlyIotHotspotV0({
+          ...args,
+          location: null,
+          elevation: 50,
+          gain: 100,
+        }).accounts({
+          rewardableEntityConfig,
+          hotspotOwner: hotspotOwner.publicKey,
+          keyToAsset,
+          iotInfo: iotInfoKey(rewardableEntityConfig, ecc)[0],
+          subDao,
+        }).signers([hotspotOwner]);
+      
+      const { iotInfo } = await onboardMethod.pubkeys();
+      await onboardMethod.rpc();
+
+      const iotInfoAccount = await hemProgram.account.iotHotspotInfoV0.fetch(
+        iotInfo!
+      );
+      expect(Boolean(iotInfoAccount)).to.be.true;
```

Comment:
Should verify information on these accounts too

Reply:
Done